src/index.html

```diff
@@ -1,12 +1,313 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet"
+          href="./styles/main.scss">
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com">
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap"
+          rel="stylesheet">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#logo"
+         class="logo">
+        <img class="logo__image"
+             src="./images/logo.png"
+             alt="two colors moyo logo">
```

Comment:
something is wrong with formatting, please try to use prettier to fix it

Reply:
I had a problems with formatting. I change it to prettier and nothing happend.